services/mailer/mail.go

```diff
@@ -68,6 +69,7 @@ func SendUserMail(language string, u *models.User, tpl base.TplName, code, subje
 
 	var content bytes.Buffer
 
+	// TODO: i18n
 	if err := bodyTemplates.ExecuteTemplate(&content, string(tpl), data); err != nil {
```

Comment:
An alternative here is to add the lang as a suffix and test for the existence of the template with that suffix before falling back to the non-suffixed variatn.